tests/lib/prepare.sh

```diff
@@ -3,7 +3,8 @@
 set -eux
 
 prepare_classic() {
-    apt install -y ${SPREAD_PATH}/../snapd_*.deb
+    # relying on dpkg as apt(-get) does not support installation from local files in trusty.
+    dpkg -i ${SPREAD_PATH}/../snapd_*.deb || apt-get -f install -y
```

Comment:
I introduced a helper conditionally calling the right cmds based on SPREAD_SYSTEM, also adjusted CUPS package installation/remove taking SPREAD_SYSTEM into account.